lib/services/runtime/handlers/utils/directives.ts

```diff
@@ -0,0 +1,25 @@
+import { Intent } from 'ask-sdk-model';
+
+import { TurnElicitSlot } from '../responseBuilders';
+
+export const createDelegateIntent = (intentName: string, slots: string[] = []): Intent => ({
+  name: intentName,
+  confirmationStatus: 'NONE',
+  slots: slots.reduce(
+    (acc, slotName) => ({
+      ...acc,
+      [slotName]: {
+        name: slotName,
+        value: '',
+        resolutions: {},
+        confirmationStatus: 'NONE',
+      },
+    }),
+    {}
```

Comment:
use `Object.fromEntries(slots.map())` instead of `slots.reduce()`

```suggestion
import { Intent, Slot } from 'ask-sdk-model';

import { TurnElicitSlot } from '../responseBuilders';

export const createDelegateIntent = (intentName: string, slots: string[] = []): Intent => ({
  name: intentName,
  confirmationStatus: 'NONE',
  slots: Object.fromEntries(
    slots.map((slotName): [string, Slot] => [
      slotName,
      {
        name: slotName,
        value: '',
        resolutions: {},
        confirmationStatus: 'NONE',
      },
    ])
```